common/reviews/api/node-library-build.api.ts

```diff
@@ -180,11 +180,13 @@ interface IFixupSettingsOptions {
   mustBeCommonJsOrEsnext: boolean;
 }
 
-// @public
+// @alpha
```

Comment:
Let's chat about this one.

Reply:
We chatted and will keep it alpha for now.

Reply:
:shipit: